dicodile/workers/dicod_worker.py

```diff
@@ -747,7 +748,7 @@ def recv_signal(self):
         local_seg_support = None
         if self.n_seg == 'auto':
             n_seg = None
-            local_seg_support = 2 * np.array(atom_support) - 1
+            local_seg_support = 2 * np.array(atom_support) + 1
```

Comment:
why `+1`? I though it was `2*atoms_support` in `alphacsc`

Reply:
But there are still other problems